docs-sources/includes/_Archivist.md

```diff
@@ -130,6 +130,13 @@ config:
 
         # optional, useful if you share a bucket with other applications
         prefix: "bob/"
+
+    # options only used with archivist:create
+    create:
```

Comment:
Not sure exactly why, I guess because options is partially feed directly to some of the couchbase functions?